scilpy/io/utils.py

```diff
@@ -146,7 +146,8 @@ def check(path):
             check(optional_file)
 
 
-def assert_outputs_exist(parser, args, required, optional=None):
+def assert_outputs_exist(parser, args, required, optional=None,
+                         check_dir_exist=False):
```

Comment:
imo it'd be better to set the default for `check_dir_exist` to True
I can't think of a use case where you would not want to make sure the directory exists beforehand

Reply:
I think the only problem is for script that migh create the folder,
However I feel most of them simply use "assert_output_dirs_exist_and_empty(...)"

Reply:
Discussed offline